Remarriage and number of FAM records (#88)
FamilySearch/GEDCOM: specification/gedcom.md

See `FAMILY_RECORD`

:::note
The common case is that each couple has one `FAM` record,
but that is not always the case.

A couple that separates and then gets together again
can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc.)
or as a separate `FAM` for each time together.
Some user interfaces may display these two in different ways
and the two admit different semantics in sourcing.

> a single `FAM` with two `MARR` with distinct dates might represent uncertainty about dates
> and a pair of `FAM` with same spouses might be the result of merging multiple files.

Author: ```suggestion
A single `FAM` with two `MARR` with distinct dates might also represent uncertainty about dates
and a pair of `FAM` with same spouses might also be the result of merging multiple files.
```

See `FAMILY_RECORD`

:::note
The common case is that each couple has one `FAM` record,
but that is not always the case.

A couple that separates and then gets together again

> can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc)

Reviewer: ```suggestion
can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc.)
```

or as a separate `FAM` for each time together.
Some user interfaces may display these two in different ways
and the two admit different semantics in sourcing.
Neither structure necessarily indicates this kind of temporary separation:
a single `FAM` with two `MARR` with distinct dates might represent uncertainty about dates

> and a pair of `FAM` with same spouses might be the result of merging multiple files.

Author: ```suggestion
A single `FAM` with two `MARR` with distinct dates might also represent uncertainty about dates
and a pair of `FAM` with same spouses might also be the result of merging multiple files.
```

Implementers should support both representations,

> and should to chose between them based on user input or other context beyond that provided in the datasets themselves.

Author: ```suggestion
and should choose between them based on user input or other context beyond that provided in the datasets themselves.
```
